TMinus/Launch.swift

```diff
@@ -29,6 +30,31 @@ struct Launch {
     //MARK: Private
 }
 
+extension Launch: NotificationType {
+    
+    func makeNotificationRequest() -> UNNotificationRequest {
+        let date = startDate.addingTimeInterval(-.oneHour)
+        let components = Calendar.current.dateComponents([.year, .month, .day, .hour, .minute, .second], from: date)
+        
+        let content = UNMutableNotificationContent()
+        var title = rocketName
+        if let missionName = missionName, !missionName.isEmpty {
+            title = "\(title) - \(missionName)"
+        }
+        content.title = title
+        content.body = "The launch window is scheduled to open in one hour."
```

Comment:
Up to you, but we may want to start localizing early. It doesn't take much investment up front, but I think the payout could be huge to internationalize this app at some point.

On some of my recent projects, we've also created a "LocalizedStrings" struct that contains sub-structs per screen to help consolidate all user-facing copy into one file and make it easier to spot duplicates. It's definitely debatable whether or not that would be overkill for such a small app or MVP though.